Detector/DetFCChhTrackerTkLayout/compact/v4.01/FCChh_v4.01.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+    <info name="TkLayoutTracker" title="TkLayoutTracker" author="Unknown author" url="http://fcc-tklayout.web.cern.ch/fcc-tklayout" status="optimization" version="FCChh_v4.01">
+        <comment>The tracker geometry as designed and optimized by tkLayout software - design version: FCChh_v4.01</comment>
```

Comment:
BTW, those the version correspond to what we have on the tklayout web site?

Reply:
yes they are!